src/plot.cpp

```diff
@@ -1582,6 +1532,273 @@ void ProjectionPlot::set_field_of_view(pugi::xml_node node)
   }
 }
 
+PhongPlot::PhongPlot(pugi::xml_node node) : RayTracePlot(node)
+{
+  set_opaque_ids(node);
+  set_diffuse_fraction(node);
+  set_light_position(node);
+}
+
+void PhongPlot::print_info() const
+{
+  fmt::print("Plot Type: Phong\n");
+  RayTracePlot::print_info();
+}
+
+void PhongPlot::create_output() const
+{
+  size_t width = pixels_[0];
+  size_t height = pixels_[1];
+  ImageData data({width, height}, not_found_);
+
+#pragma omp parallel
```

Comment:
Can we use a regular `#pragma omp for` here?

Reply:
yes, for sure! I'll change it to use that instead.